client/src/stories/Input.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import InputField, { InputContainer } from '../components/Input';
+import EmailAtIcon from '../assets/email-at.svg';
+import PasswordIcon from '../assets/password.svg';
+import UserIcon from '../assets/user.svg';
+import LocationIcon from '../assets/pin.svg';
+import AgeIcon from '../assets/age.svg';
+import EmailIcon from '../assets/email.svg';
+import PhoneIcon from '../assets/phone.svg';
+import UserGroupIcon from '../assets/user-group.svg';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Inputs',
+};
+
+export const mailInput = () => (
+  <InputContainer>
+    <img src={EmailAtIcon} alt="email" />
```

Comment:
You should overthink your `alt` description.
The `alt` is used to give blind people a hint, what the image looks like. I don't know the EmailAtIcon or the PasswordIcon and I have no idea how they look like just on your `alt` description.

Reply:
You are right when I think that the placeholder text explains what kind of input field it is, here the old is actually double 😅 Thanks for the hint! :)